test/run_tests.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+BASEDIR=$(realpath $(dirname `[[ -L "$0" ]] && readlink -f "$0" || echo "$0"`)/..)
+TESTDIR="$BASEDIR/test"
+
+export PYTHONPATH="$SC:$SC/python:$BASEDIR/:$BASEDIR/sub/util"
+for f in $(ls "$TESTDIR"/test*.py); do 
+    echo "=============== $f"
+    python3 "$f"
+done
```

Comment:
Hey @juagargi, thanks for the test runner script. As discussed, my feeling is that this messing with the PYTHONPATH *should* not be necessary, and I hope this will (partially) be fixed by merging the repositories.

Either way, the bash-loop can be avoided by using e.g. pythons "native" test discovery:
```bash
PYTHONPATH="$SC:$SC/python:$BASEDIR/sub/util" python3 -m unittest discover $TESTDIR
```
I'll make this change if that seems ok for you.